DOCS-2886: Update installation instructions to separate OS install (#3477)
viamrobotics/docs: docs/tutorials/control/air-quality-fleet.md

1. Connect your first single-board computer to power.
   For this tutorial, we'll treat this as the machine for our first customer, Antonia.

>    If the computer does not already have a Viam-compatible operating system installed, follow the [Prepare your board steps in the Installation Guide](/installation/#platform-requirements) to install a compatible operating system.

Reviewer: ```suggestion
   If the computer does not already have a Viam-compatible operating system installed, follow the [Platform Requirements section of the Installation Guide](/installation/#platform-requirements) to install a compatible operating system.
```
